src/core/credentials/transport/google_default/google_default_credentials.cc

```diff
@@ -440,12 +406,15 @@ grpc_channel_credentials* grpc_google_default_credentials_create(
       call_creds_for_tls);
   grpc_error_handle error;
   grpc_core::ExecCtx exec_ctx;
+  DefaultCallCredentialsCreationMethod default_credentials_type =
```

Comment:
We don't use it here. So it's better to pass nullptr and do null checks in the function.